test/screens/play/card_viewer_test.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:treasure_of_the_high_seas/model/card/basic/a_game_of_cards.dart';
+import 'package:treasure_of_the_high_seas/model/card/basic/a_rival_ship.dart';
+import 'package:treasure_of_the_high_seas/model/card/basic/an_island.dart';
+import 'package:treasure_of_the_high_seas/model/card/card.dart' as Model;
+import 'package:treasure_of_the_high_seas/screens/play/card_viewer.dart';
+
+import '../../test_utils.dart';
+
+void main() {
+
+  List<Widget> getButtons(context, card) {
+    return [Text("Test Button 1"), Text("Test Button 2")];
```

Comment:
I guess we could prove that the `card` argument is being passed correctly if we made use of it here. E.g. we could set button text to "Button 1 for card ${card.name}" or whatever, and check that works :shrug:

Reply:
Sure, I'll stick that in later on tonight.